public/app/core/components/NavBar/NavBar.tsx

```diff
@@ -74,6 +79,8 @@ export const NavBar: FC = React.memo(() => {
             key={`${link.id}-${index}`}
             isActive={isMatchOrChildMatch(link, activeItem)}
             link={{ ...link, subTitle: undefined, onClick: undefined }}
+            showMenu={!link.highlighted}
```

Comment:
It means that we don't show the menu for highlighted elements, since those are likely to be enterprise features, not enabled for a customer. The naming could probably use some work 🤔